### DISCUSSION TITLE
use `<percentage>` for alpha channel in `toString()`?

### DISCUSSION THREAD

**Discussion by stefanprobst** [color-js/color.js#334] [Q&A] [answered]
hi, is it currently possible to serialize the alpha channel to a percentage value?

i tried `color.toString({ name: "hsl", coords: ["<angle>", "<percentage>", "<percentage>"], alpha: "<percentage>" })` but that does not seem to work.

**Comment by LeaVerou** [comment, marked as answer]
Oof, unfortunately no, you can only customize how coordinates are serialized but alpha is always a number. That's a pretty big oversight on our part, I'll convert this to an issue so we can implement it, but need to think of the best syntax to expose this nicely.

**Comment by svgeesus** [comment]
Color.js serializes `alpha` as a number because CSS Color 4 [says, in  Serializing alpha values](https://drafts.csswg.org/css-color-4/#serializing-alpha-values):

> Otherwise, return the result of serializing the given value (as a [&lt;number>](https://drafts.csswg.org/css-values-4/#number-value), not a [&lt;percentage>](https://drafts.csswg.org/css-values-4/#percentage-value)).

So &lt;percentage> is accepted on input but not generated on output.
